### PR TITLE
Extra nuclear data libs

### PR DESCRIPTION
**Pull request by Shimwell**
Hi all,

Here is a pull request for some scripts that ease the downloading of FENDL and TENDL nuclear data libraries for your consideration.

The scripts are based on the existing OpenMC scripts for downloading JEFF and NNDC data. These two evaluations are of particular interest for fusion simulations as the ITER reactor requests FENDL  data is used and the transmutation we expect in fusion reactors requires a library like TENDL with lots of isotopes when simulating burn-up.

Eventually I would like to add options for different releases, file formats (ENDF or ACE) and further nuclear data libraries such as CENDL and JENDL but I wanted to see how this first nuclear data pull request is received before going to far. I think some users will be keen on downloading nuclear data before it is converted to h5 format in case extra processing is required (e.g derived MT numbers)

### CODE REVIEW

File: scripts/openmc-get-tendl-2015-data [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import glob
+import argparse
+from string import digits
+from urllib.request import urlopen
+
+import openmc.data
+
+
+description = """
+Download TENDL 2017 ACE data from PSI and convert it to a HDF5 library for
```
Comment (makeclean) [inline scripts/openmc-get-tendl-2015-data]: script name says 2015 but says 2017 in the description?

**Comment by paulromano** [comment]
Hi @Shimwell, and thanks for this contribution! One thing I've been thinking recently is that perhaps we ought to separate out these data generation scripts into a separate repository altogether (openmc-dev/data?). Since we are planning on distributing HDF5 libraries directly, my hope is that most users won't ever need to mess around with running these scripts; they can just download the library and be on their merry way. Of course for developers, it's useful to have these scripts around whenever we want to regenerate things or see how they were generated in the first place. My feeling is that we shouldn't pollute the main openmc repository with more and more of these "get-<somelibrary>-data" scripts and instead maintain them in a separate repo.

Other thoughts @openmc-dev/committers?

**Comment by Shimwell** [comment]
Hi Paul

Yes I can see it might get cluttered with too many get--data scripts. 

What do you think of bundling all the download nuclear data scripts could into one script which takes the library and revision as arguments.

By default this could be the NNDC nuclear data but optionally someone could specify TENDL 2017, FENDL 3.1b or others.

I have just rewritten the two TENDL 2015 and TENDL 2017 scripts so that there is now one TENDL script. Which allows both releases to be downloaded. So there is now one less openmc-get* script and a little less clutter.

Of course a separate repo (within openmc-dev) is also a good idea. I am just keen to see other libraries so I will be keen to help with whatever option works best for you.

**Comment by Shimwell** [comment]
I have now made the openmc-get-fendl-script capable of downloading different releases of fendl. So there would be no need to have separate scripts for each release. 

So this pull request is proposing that openmc-get-*-data scripts are made for the major libraries and support for new releases of those libraries as they arrive.

The current state is that we have jeff and nndc, this PR could add tendl and fendl then a later PR can add cendl and jendl. Let me know if I have missed any

openmc-get-**nndc**-data (script complete for 1 release of ENDF)
openmc-get-**jeff**-data (script complete for 1 release of jeff)
openmc-get-**fendl**-data (script complete for 2 release of fendl)
openmc-get-**tendl**-data (script complete for 2 release of tendl) 
openmc-get-**cendl**-data (script proposed in another PR) 
openmc-get-**jendl**-data (script proposed in another PR) 

Perhaps in the future argument options can be added for those who want to download ACE, ENDF or H5 format and the script carries out the necessary steps to convert the ACE->H5 or the ENDF->ACE->H5 or just downloads the H5 (which can be the default option) and makes the cross_sections.xml . 

Keen to hear peoples thoughts, best Jon

**Comment by paulromano** [comment]
Thanks @Shimwell. An update on my end -- I've created a new [openmc-dev/data](https://github.com/openmc-dev/data) repository that has already been populated with some of the scripts. I've modernized three of the scripts (all the LANL-based libraries) while I was at it. The libraries that I've produced from those scripts are now available at https://openmc.mcs.anl.gov. I'm going to go ahead and close this PR and then you can reopen one on the new repository.

**Pull request closed without merging**